concierge_scheduler/concierge_zabbix.py

```diff
@@ -221,22 +252,45 @@ def get_simple_id_map(self):
         we need to have a simplified map of component IDs to their names because
         when we're importing things like auto-registration actions, we need to
         know what our old and new IDs are so we can correctly link them upon
-        import
+        import.
+        The hash of the components data (excluding id) is also stored for comparing data.
 
         :return: file
         """
         data = defaultdict(list)
 
         for template in self.zbx_client.template.get(output="extend"):
+            template_data = dict(template)
+            del template_data['templateid']
+            template_hash = hashlib.md5(json.dumps(template_data, sort_keys=True).encode("utf-8")).hexdigest()
             templates = \
```

Comment:
why is this line not just performed as part of the next line? (Also ctrl-alt-l) your files to auto format becaues you've a lot of lines longer than PEP standards)

Reply:
Sorry which line is it you are talking about? I had it separate for readability. I have formatted the files

Reply:
Line 231. Totally for readability, it's just that I don't think it really adds anything here. If there was some list  comprehension, creating a new set of templates, then it makes sense but here it's a single dictionary